library/src/morpheus-master/sass/modules/tool/commons/_commons.scss

```diff
@@ -245,3 +245,9 @@
         padding-right: 3px;
     }
 }
+
+#commons-imagefile-instruction {
+    font-weight: bold;
+    font-size: bigger;
```

Comment:
is "bigger" valid ?  

https://developer.mozilla.org/en-US/docs/Web/CSS/font-size

Reply:
No. It should be larger. Bigly larger.